index.js

```diff
@@ -113,7 +108,14 @@ module.exports = {
 		'no-var': 'warn',
 		'require-yield': 'off',
 		'use-isnan': 'warn',
-		'wrap-iife': ['error', 'inside'],
+
+		// @stylistic/js plugin https://github.com/eslint-stylistic/eslint-stylistic
+		'@stylistic/js/new-parens': 'warn',
+		'@stylistic/js/no-extra-semi': 'off',
```

Comment:
When checking the following rules, the result of the `enact lint` differ between develop branch and the feature branch.
'@stylistic/js/no-extra-semi': 'off'
'@stylistic/js/no-mixed-spaces-and-tabs': ['warn', 'smart-tabs'].
Could you please check it?

Reply:
When checking the commit below, these two rules still exist in eslint v8 and will be deprecated starting from eslint v9(https://github.com/eslint/eslint/commit/6ee3e9eb5df7bdfdaa1746214793ed511112be76).
In this PR, I will restore the code.